src/dataflow/src/render/mod.rs

```diff
@@ -714,7 +714,14 @@ where
                 Some(button)
             }
             SinkConnector::Tail(c) => {
-                sink::tail(&collection.inner, sink_id, c);
+                // Map by sink_id is not needed for correctness, but will spread the work
+                // around when there are multiple TAILs running.
+                // Arranging will cause updates to be presented in time order.
+                let stream = collection
```

Comment:
just checking my understanding: mapping by sink id will move everything to one worker right?

Reply:
I'm asking because it seems like having everything on one worker seems necessary for correctly emitting records in ts order given the tail implementation in this pr, but if not, then I'm definitely missing something

Reply:
My understanding is that your understanding is correct!

Reply:
I think the intent of the comment might be that "mapping by _sink id_ specifically is not required, and you could map row to (0, row) instead and still be correct"

Reply:
I've improved the comment here to try to make this more clear.